api/proto/info/info.proto

```diff
@@ -278,6 +278,20 @@ message ZCellularModuleInfo {
     string firmware_version = 3;
 }
 
+// Device Run State
+enum ZDeviceRunState {
+    // Just to handle Un-Initialized runState variable Internally
+    ZDEVICE_RUN_STATE_UNSPECIFIED = 0;
+    ZDEVICE_RUN_STATE_ONLINE = 1;
+    // Device Reboot in Progress. During bootup, Device reports "BOOTING"
+    ZDEVICE_RUN_STATE_REBOOTING = 2;
+    ZDEVICE_RUN_STATE_MAINTENANCE_MODE = 3;
+    // Device BaseOs Upgrade in Progress
+    ZDEVICE_RUN_STATE_BASEOS_UPGRADING = 4;
+    // Device is booting up
```

Comment:
Presumably this will be set until the initial set of objects have been processed. But I assume the app instances might not have completed their boot when the device transitions from BOOTING to ONLINE. Please document.

Reply:
Clarified that for ONLINE:

    // Device is Online. This only means all components of EVE are up.
    // This does not indicate the Status of Apps or other configured objects.
    // During booting up after reboot, Device might be reported as ONLINE
    // while apps are still being processed.
ZDEVICE_STATE_ONLINE = 1;

Changed Description of BOOTING to:

    // Device is booting up. This state is reported when EVE is started and
    // is able to communicate with the controller. When all the components of
    // EVE are up, it transitions to ONLINE.
    ZDEVICE_STATE_BOOTING = 5;